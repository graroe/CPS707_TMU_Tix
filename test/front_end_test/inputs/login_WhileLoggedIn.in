login
sales
login
logout
login
admin
login
logout