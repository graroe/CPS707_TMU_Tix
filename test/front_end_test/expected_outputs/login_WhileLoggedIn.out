Enter session type <sales or admin>: 
Sales agent login successful. Already logged in.
Session terminated.
Enter session type <sales or admin>: 
Admin login successful.
Already logged in.
Session terminated.